src/PDFtoImage/PdfiumViewer/NativeMethods.cs

```diff
@@ -15,6 +15,8 @@ static NativeMethods()
                 Assembly.GetExecutingAssembly().GetName(false).CodeBase
                 ?? Process.GetCurrentProcess().MainModule!.FileName!;
 
+            workingDirectory = workingDirectory ?? AppContext.BaseDirectory;
```

Comment:
Could you please change it into this (just for code styling):

```csharp
var workingDirectory =
	Assembly.GetExecutingAssembly().GetName(false).CodeBase
	?? Process.GetCurrentProcess().MainModule?.FileName
	?? AppContext.BaseDirectory;
```

Other than that it seems to work perfectly fine!